core/node/fee_model/src/l1_gas_price/gas_adjuster/mod.rs

```diff
@@ -200,6 +200,11 @@ impl GasAdjuster {
             PubdataSendingMode::Calldata => {
                 self.estimate_effective_gas_price() * self.pubdata_byte_gas()
             }
+            PubdataSendingMode::Custom => {
+                // Fix this when we have a better understanding of dynamic pricing for custom DA layers.
+                // GitHub issue: https://github.com/matter-labs/zksync-era/issues/2105
+                0
+            }
```

Comment:
that's a valid point, it might make sense for some operators to set at least some constant value there, to cover some percentage of the costs
but let's keep this feature for a separate PR to not inflate this one